File: 단계별로_풀어보기/2. 조건문/4. 14681.js
const fs = require('fs')
const inputs = fs.readFileSync(0, 'utf-8').toString().split('\n')
// const inputs = fs.readFileSync('/dev/stdin').toString().split('\n')

const [a, b] = inputs

const A = parseInt(a)
const B = parseInt(b)

let result = ''

if (A > 0) {
  if (B > 0) {
    result = '1'
  } // ::
  else {
    result = '4'
  }
} // ::
else {
  if (B > 0) {
    result = '2'
  } // ::
  else {
    result = '3'
  }
}

process.stdout.write(result)

File: 1. 단계별로_풀어보기/1~9/2. 조건문/4. 14681.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

X = int(input())
Y = int(input())

xx = 1.5 if X > 0 else 0.5
yy = -1 if Y > 0 else 1

result = int(2.5 + xx * yy)

output(f"{result}")